packages/react/src/NavList/NavList.tsx

```diff
@@ -33,7 +33,7 @@ const NavBox = styled.nav<SxProp>(sx)
 const Root = React.forwardRef<HTMLElement, NavListProps>(({children, ...props}, ref) => {
   return (
     <NavBox {...props} ref={ref}>
-      <ActionList>{children}</ActionList>
+      <ActionList role="list">{children}</ActionList>
```

Comment:
Yup, exactly! I used it here as an "escape-hatch" so that `NavList`, (and potentially others) could keep the default `ActionList` semantics.

Reply:
Can you say more, what are the default ActionList semantics?

Reply:
The default ActionList semantics would be `<li><button>Item</button></li>` with any props being placed on the button rather than the list item. The criteria to maintain the same semantics (i.e. only `<li>`, no `<button>`) are:

* If either ARIA role is provided (`role="list"`, `role="listbox"`, `role="menu"`)
* If `inactive` is passed to `ActionList.Item`
* If a valid value of `_PrivateItemWrapper` is passed
* If `ActionList` is used within an `ActionMenu`

So we'd have this (default in this PR):

```
<ul>
  <li>
    <button {...props}>Item 1</button>
  </li>
</ul>
```

and the semantics if any of the criteria is met (default currently outside of this PR):
```
<ul>
  <li tabindex="0" {...props}>
    item 1
  </li>
</ul>
```